packages/react-components/src/components/MessageThread.test.tsx

```diff
@@ -8,13 +8,16 @@ import { MessageThread } from './MessageThread';
 import { ChatMessage } from '../types';
 /* @conditional-compile-remove(data-loss-prevention) */
 import { BlockedMessage } from '../types';
+/* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
 import { AttachmentDownloadResult, AttachmentMetadata } from './FileDownloadCards';
 import { createTestLocale, renderWithLocalization } from './utils/testUtils';
 /* @conditional-compile-remove(date-time-customization) @conditional-compile-remove(data-loss-prevention) */
 import { COMPONENT_LOCALE_EN_US } from '../localization/locales';
 /* @conditional-compile-remove(date-time-customization) */
 import { screen } from '@testing-library/react';
+/* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
 import { render, waitFor } from '@testing-library/react';
+/* @conditional-compile-remove(data-loss-prevention) */ /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */

```

Comment:
Do we need to have /* @conditional-compile-remove(data-loss-prevention) */ here?

Reply:
We need the data-loss prevention one because it's used in line 145